CONTRIBUTE.md

```diff
@@ -42,12 +42,15 @@ Run the commands (or follow the MkDocs documentation to locally pip install MkDo
 
     # requirements.txt
 
-    ## using pip
-    pip install -r requirements.txt
+    ## using pip and venv
+    python -m venv venv
+    source venv/Scripts/activate
```

Comment:
I think this should be `source venv/bin/activate` for linux, or `venv\Scripts\activate` for Windows.

Reply:
ooh yes please that'd be great, thanks!